tests/saitests/py3/sai_qos_tests.py

```diff
@@ -351,6 +351,7 @@ def runTest(self):
         dual_tor = self.test_params.get('dual_tor', None)
         leaf_downstream = self.test_params.get('leaf_downstream', None)
         asic_type = self.test_params['sonic_asic_type']
+        platform_asic = self.test_params['platform_asic']
```

Comment:
Please revert as we are fixing the queueu swap issue.

Reply:
Reverted the commit for DscpQueueMapping test